.github/workflows/nightly-benchmark-overhead.yml

```diff
@@ -1,4 +1,4 @@
-name: benchmark-overhead tests
+name: Nightly overhead benchmark
```

Comment:
does this generate a report on github somewhere?

Reply:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/gh-pages/benchmark-overhead/results